### PR TITLE
Fix hardcoded configuration for create collection call

### PR DESCRIPTION
**Pull request by bibliotechy**

### CODE REVIEW

File: tulflow/solr_api_utils.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def delete_collection(self, collection):
         else:
             logging.info(f"Collection {collection} did not exist. Continuing without error")
 
-    def create_collection(self, collection, configset="_default" ,numShards=1, replicationFactor=1, maxShardsPerNode=1 ):
-        path = f"/solr/admin/collections?action=CREATE&name={collection}&collection.configSet=configset&numShards=1&replicationFactor=1&maxShardsPerNode=1"
```
Comment (cmharlow) [inline tulflow/solr_api_utils.py]: do we want replicationFactor hardcoded?

Comment (bibliotechy) [reply tulflow/solr_api_utils.py]: ugh, no

Comment (bibliotechy) [reply tulflow/solr_api_utils.py]: Do we? Seems like we might want to be flexible, right?

Comment (cmharlow) [reply tulflow/solr_api_utils.py]: yeah, thought so, but saw you changed it here to being hard-coded, hence asking

Comment (cmharlow) [reply tulflow/solr_api_utils.py]: there's a shit ton of linting / syntax errors, so i'm just gonna push that up to this branch while chcking the replication var

**Pull request merged**